frontend/src/libraries/explorer-wamp/nodes.ts

```diff
@@ -39,22 +40,34 @@ export default class NodesApi extends ExplorerApi {
       whereClause += ` AND is_validator = 0 `;
     }
     try {
-      const nodes = await this.call<NodeInfo[]>("select", [
-        `SELECT ip_address as ipAddress, moniker, account_id as accountId, node_id as nodeId, signature, 
-        last_seen as lastSeen, last_height as lastHeight, last_hash as lastHash,
-        agent_name as agentName, agent_version as agentVersion, agent_build as agentBuild,
-        peer_count as peerCount, is_validator as isValidator, status
-            FROM nodes
-            ${whereClause}
-            ORDER BY node_id DESC
-            LIMIT :limit
-        `,
-        {
-          limit,
-          paginationIndexer,
-          validatorIndicator,
-        },
+      const [nodes, validators] = await Promise.all([
+        this.call<NodeInfo[]>("select", [
+          `SELECT ip_address as ipAddress, moniker, account_id as accountId, node_id as nodeId, signature, 
+          last_seen as lastSeen, last_height as lastHeight, last_hash as lastHash,
+          agent_name as agentName, agent_version as agentVersion, agent_build as agentBuild,
+          peer_count as peerCount, is_validator as isValidator, status
+              FROM nodes
+              ${whereClause}
+              ORDER BY node_id DESC
+              LIMIT :limit
+          `,
+          {
+            limit,
+            paginationIndexer,
+            validatorIndicator,
+          },
+        ]),
+        this.queryValidators(),
```

Comment:
Let's decouple fetching the list of nodes based on telemetry from the RPC query. Nodes List page is quite important and we should not block the display when RPC is unreachable.